openedx/frontend-app-authoring
# feat: add component sidebar manage tab [FC-0062]

## Description

This PR adds the `Manage` tab in the Component Sidebar.

![image](https://github.com/user-attachments/assets/b150e0c6-6e14-4862-895b-7be7d7524bb5)

## More information

Part of:
 - https://github.com/openedx/frontend-app-course-authoring/issues/1190

Depends on:
- https://github.com/openedx/edx-platform/pull/35476

## Testing Instructions

- Checkout this branch and https://github.com/openedx/edx-platform/pull/35476
- Create a new library and add a component to it
- Click on the component and check the draft status on the `Manage` tab
- Publish the library
- Go back to the company and check the published status on the `Manag` tab
- 
___
Private ref: [FAL-3802](https://tasks.opencraft.com/browse/FAL-3802)
